common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -181,6 +181,10 @@ export interface ISignalMessage {
     clientId: string | null;
 
     content: any;
+
+    signalSequenceNumber?: number;
```

Comment:
should be call this clientSequenceNumber, same as ops. or is that confusing? this number is should be the number of signals sent by that clientid

Reply:
ideas: clientSignalNumber, clientSignalCount, clientSingalIndex

Reply:
I like `clientSequenceNumber`. We can document that it's specifically a "sequence number" purely related to signals.
any thoughts @vladsud @tanviraumi ?

Reply:
Given that e (at the moment) do not have a plan to extend it to all signals, I think I like Tanvir's direction of thinking more than other directions

Reply:
Sounds good. I changed it to `clientConnectionNumber`